package.json

```diff
@@ -45,6 +45,7 @@
     "osenv": "^0.1.5",
     "read-package-json": "^2.1.0",
     "readable-stream": "^3.4.0",
+    "resolve": "^1.13.1",
```

Comment:
you can remove resolve from here. No need to install it if we don't use it for the source code of citgm itself.